kbs/Makefile

```diff
@@ -20,6 +21,14 @@ else
   AS_FEATURE = $(AS_TYPE)
 endif
 
+ifndef CLI_FEATURES
+  ifdef ATTESTER
+    CLI_FEATURES = "sample_only,$(ATTESTER)"
```

Comment:
It's a bit misleading to use both `sample_only` and another attester, but I guess we can live with that.